src/ui/common/src/handlers/NodeOperatorGet.ts

```diff
@@ -0,0 +1,18 @@
+// This file should map exactly to
+// src/golang/cmd/server/handler/v2/dag_get.go
```

Comment:
Is there a reason why this file and the NodesGet.ts file below are two separate files?

If we want them both to map to dag_get.go, should we consider combining them all into one?

Reply:
ohhh that's a typo in comment. They should map to different `.go` files